### ISSUE TITLE
Error on latex output.

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#769]
enviroment:
- conf.py langage='ja'
- Sphinx: 1.1pre (af21bf194936)
- docutils 0.8.1

Sphinx log

```
! Package babel Error: You haven't loaded the option english yet.
See the babel package documentation for explanation.
Type  H <return>  for immediate help.
...
l.18 \select@language{english}
)
```

Works fine for "Docutils version: 0.7".

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/769
- Originally reported by: Anonymous
- Originally created at: 2011-09-22T16:15:40.239

**Comment by shimizukawa** [comment]
_From Anonymous on 2011-09-22 14:28:21+00:00_

enviroment
- conf.py language='ja' only
- Sphinx 1.1pre (af21bf194936)
- docutils 0.8.1

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-22 16:13:58+00:00_

fixed with PR#23.

**Comment by shimizukawa** [comment]
_From Andi Albrecht on 2012-02-14 09:03:47+00:00_

FTR, I've seen the same issue after changing the language in conf.py. Running "make clean" solved it for me.
